### PR TITLE
Removed links to internal resources

### PR DESCRIPTION
**Pull request by dianashevchenko**
References PRs #1350 and #1346 

- multipleIPHeaders is documented to be used internally, so I removed it from ext/tags.go list of public API tags
- removed links to internal resources 

TODO:
- add a page of public documentation on DD_TRACE_HEADER_TAGS at https://docs.datadoghq.com/tracing/trace_collection/library_config/go/ or https://docs.datadoghq.com/tracing/trace_collection/tracing_naming_convention/#http-requests

### CODE REVIEW

**Comment by dianashevchenko** [comment]
cc @Hellzy

**Review by ajgajg1134 — APPROVED** [review]
Looks good!

File: contrib/internal/httptrace/config.go [hunk 1]
```diff
@@ -27,7 +27,6 @@ const (
 )
 
 // defaultQueryStringRegexp is the regexp used for query string obfuscation if `envQueryStringRegexp` is empty.
-// The regexp is taken from https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2490990623/QueryString+-+Sensitive+Data+Obfuscation
```
Comment (Hellzy) [inline contrib/internal/httptrace/config.go:30]: We could link to https://docs.datadoghq.com/tracing/configure_data_security#redacting-the-query-in-the-url instead.

File: contrib/internal/httptrace/httptrace.go [hunk 2]
```diff
@@ -157,7 +160,6 @@ func isGlobal(ip netaddr.IP) bool {
 
 // urlFromRequest returns the full URL from the HTTP request. If query params are collected, they are obfuscated granted
 // obfuscation is not disabled by the user (through DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP)
-// For more information see https://datadoghq.atlassian.net/wiki/spaces/APM/pages/2357395856/Span+attributes#http.url
```
Comment (Hellzy) [inline contrib/internal/httptrace/httptrace.go:160]: Same here, a link to https://docs.datadoghq.com/tracing/configure_data_security#redacting-the-query-in-the-url wouldn't hurt.

File: contrib/internal/httptrace/httptrace.go [hunk 3]
```diff
@@ -91,7 +95,6 @@ func ippref(s string) *netaddr.IPPrefix {
 }
 
 // genClientIPSpanTags generates the client IP related tags that need to be added to the span.
-// See https://datadoghq.atlassian.net/wiki/spaces/APS/pages/2118779066/Client+IP+addresses+resolution
```
Comment (Hellzy) [inline contrib/internal/httptrace/httptrace.go:94]: Public doc for this: https://docs.datadoghq.com/tracing/configure_data_security#configuring-a-client-ip-header

**Review by Hellzy — COMMENTED** [review]
Thanks for the cleanup Diana 👍

**Review by ajgajg1134 — APPROVED** [review]

**Pull request merged**